fMRI_Tutorial/DataCheck.ipynb

```diff
@@ -0,0 +1,523 @@
+      "text/plain": [
+       "<Figure size 1440x1080 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "matdropped = df.dropna(axis=1, how='all').dropna(axis=0,how='all').copy()\n",
+    "plt.figure(figsize=(20,15))\n",
+    "ax = sns.heatmap(matdropped, cmap=pallete, annot=False)\n",
+    "labels = [value.get_text() for value in list(ax.get_xticklabels())]\n",
+    "labelsy = [value.get_text() for value in list(ax.get_yticklabels())]\n",
+    "ax.set_yticklabels(pd.DataFrame(names).iloc[labelsy][0], rotation=0)\n",
+    "ax.set_xticklabels(pd.DataFrame(names).iloc[labels][0])\n",
+    "ax.set_xlim(0, len(matdropped))\n",
+    "ax.set_ylim(len(matdropped), 0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, one does not see these blank spaces anymore. One can also check how the matrice will not have NaNs anymore. \n",
+    "As discussed above, it is possible to see 3 clear diagonals in the matrix. \n",
+    "\n",
+    "\n",
+    "##### Now let's plot the connectivity distribution plots.\n",
+    "The distribution of connectivity easily shows whether there are outliers in connectivity values in your matrix, and whether the overall distribution fits with what we expect of fMRI data (i.e. a right-skewed distribution)."
```

Comment:
The diagonal is set to 1, which means that there are ~78 outliers in the connectivity distribution. Would it make more sense to set the diagonal to 0 for displaying purposes? In general, I do set the diag to 0 (and you set it to NaN later), but I get that you want to check for 0's earlier in the code...

Reply:
I sent it to 0 just for the distribution plot. I believe that sending it to all plots before normalization will make the diagonal too 'in-between' values/colors and hide it a bit.

Reply:
Because this PR was open, the adding of the Preproc scripts fell into here too. Just to let you know :)

Reply:
@lindadouw I think this PR is finished by now and could be closed/merged. I've changed the things you asked, added the DMN things, if you want to have a final look :)